src/vert_initialize.lua

```diff
@@ -75,8 +76,8 @@ function M.init(opts)
 
   deactivate nondestructive
 
-  LUA_VERSION="%s"
-  VERT="%s"
+  LUA_VERSION="%{lua_version}"
+  VERT="%{prefix}"
```

Comment:
I think a syntax like this for the templates is nicer than using the `%s` we had before.